actors/builtin/paych/paych_test.go

```diff
@@ -29,19 +29,20 @@ func TestExports(t *testing.T) {
 
 func TestPaymentChannelActor_Constructor(t *testing.T) {
 	ctx := context.Background()
-	actor := pcActorHarness{Actor{}, t}
-
 	paychAddr := tutil.NewIDAddr(t, 100)
 	payerAddr := tutil.NewIDAddr(t, 101)
+	payeeAddr := tutil.NewIDAddr(t, 102)
 	callerAddr := tutil.NewIDAddr(t, 102)
 
+	actor := pcActorHarness{Actor{}, t, paychAddr, payerAddr, payeeAddr}
+
 	t.Run("can create a payment channel actor", func(t *testing.T) {
 		builder := mock.NewBuilder(ctx, paychAddr).
 			WithCaller(callerAddr, builtin.InitActorCodeID).
-			WithActorType(paychAddr, builtin.AccountActorCodeID).
```

Comment:
This was setting the payment channel actor as having an account actor code CID 😕. I think due to (ab)using the actor as the channel's payee, which is invalid and only worked here due to hacking the runtime like this.